### PR TITLE
Fix : final fixes

### PR DESCRIPTION
**Pull request by poornima-alugubelly**
Hey guys I have made some  final fixes to the CL 
-Added Correct carbon snippets
- Added floating button
- Fixed links

### CODE REVIEW

File: components/Navigation/Navigation.css [hunk 1]
```diff
@@ -3,3 +3,134 @@
...
+	background-color: var(--primary-color);
+	display: block;
+}
+/* ------------------- */
+/* --------DRAWER--------------- */
+
+.drawer {
+	width: fit-content;
+	text-align: center;
+	background-color: var(--black);
```
Comment (kiranpratyush) [inline components/Navigation/Navigation.css:99]: Just a suggestion We can change the name to --color-black

File: components/Navigation/Navigation.css [hunk 2]
```diff
@@ -3,3 +3,134 @@
...
+	font-family: var(--secondary-font);
+	color: var(--secondary-color);
+	font-size: inherit;
+	display: flex;
+}
+
+.drop-down {
+	padding: 1rem;
+	position: absolute;
+	top: 100%;
```
Comment (kiranpratyush) [inline components/Navigation/Navigation.css]: We can use bottom 0. Correct me If I am wrong

**Review by kiranpratyush — APPROVED** [review]
We can use prettier (something like that) to format it beautifully

File: components/Navigation/Navigation.css (hunk 2 above)
Comment (poornima-alugubelly) [reply components/Navigation/Navigation.css]: oh yes u r right! that is actually redundant code I have removed it now thanks for pointing it out!

**Review by MiheerTamkhane — APPROVED** [review]
image
I think there are too many files thats why I skipped through html part and other than that all good I think. Also in UI your sidebar is overflowing try to correct that.

**Review by msdeshmukh009 — APPROVED** [review]
The code is looking good and maybe add smooth scroll behavior so there will be smooth scrolling if somebody clicks on the floating action button.
```scroll-behavior: smooth;```

**Pull request merged**